### DISCUSSION TITLE
Search function for emotes?

### DISCUSSION THREAD

**Discussion by mustek426** [bttv-android/bttv#466] [Ideas]
Is it possible to add a search function for emotes? The extentions have loads of emotes and being subscribed to big channels that also has loads of emotes makes scrolling through all of them to find the one I want to use a hassle and takes a while.

**Comment by FoseFx** [comment]
You can use the ":" (colon) to open an Emote search pop-up. This is a native Twitch feature bttv-android supports
![Screenshot_2022-06-04-20-49-49-84_34750238b5877b2a91c912748cb283f8.jpg](https://user-images.githubusercontent.com/22842759/172021624-41bbb5ce-26dc-45dc-8b5f-d7712e0486a9.jpg)

**Reply by mustek426** [reply]
Oh okay didn't know that thanks, I would always use the search on the 7tv emote browser.
